inspirehep/modules/orcid/static/js/inspire_profile_matcher/templates/match_profile.html

```diff
@@ -0,0 +1,84 @@
+<form  ng-controller="inspireProfileMatcherCtrl" ng-submit="postMatches()">
+
+  <div id="collection-landing-page-text">
+    <div class="row">
+      <div class="col-xs-12">
+          <p class="h3 text-center">
```

Comment:
Where is the `h3` class defined? It does not seem a good name to style a `p` element.